edx_notifications/server/api/consumer.py

```diff
@@ -117,7 +118,7 @@ def get(self, request):
             return Response({}, status.HTTP_400_BAD_REQUEST)
 
         cnt = get_notifications_count_for_user(
-            request.user.id,
+            int(request.user.id),
```

Comment:
Sorry, no dice. My unit tests are now saying:
ERROR: Failure: ContractSyntaxError (Unknown identifier 'long'. Did you mean 'None'? (at char 4), (line:1, col:5)

 line  1 >int|long,>0
              ^
              |
              here or nearby)

'long' doesn't seem to be a contract keyword...